css/logo-nav.css

```diff
@@ -739,15 +739,15 @@ a {
     content:"";
 }
 
-@media (min-width: 510px) {
+@media (min-width: 710px) {
```

Comment:
why change this one to 710 ? We already have a transition at 768, can't we use that one ?